astro.js

```diff
@@ -0,0 +1,18 @@
+/** @type {import("eslint").Linter.Config}  */
+module.exports = {
+  extends: ['@sparing-software/eslint-config', 'plugin:astro/recommended'],
```

Comment:
Is it possible to put our configuration at the end?

Reply:
Yes, we can do it :)